python/change-notes/2021-02-25-port-stactrace-exposure-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Ported URL redirection (`py/stack-trace-exposure`) query to use new data-flow library. This might result in different results, but overall a more robust and accurate analysis.
```

Comment:
Following [suggestion](https://github.com/github/codeql/pull/5075#discussion_r580953866) from `@felicitymay` in one of my other PRs.

I also just noticed now that the title said `URL redirection` :laughing: 

```suggestion
lgtm,codescanning
* Updated _Information exposure through an exception_ (`py/stack-trace-exposure`) query to use the new data-flow library and type-tracking approach instead of points-to analysis. You may see differences in the results found by the query, but overall this change should result in a more robust and accurate analysis.
```